deploy/gcloud_env_vars.sh

```diff
@@ -23,6 +23,8 @@ export GITHUB_MAIN_BRANCH_NAME=main
 export PROJECT_IS_USING_WHITENOISE=true
 
 export TEST_COVERAGE_BUCKET_NAME=hopic-test-coverage-reports
+export TEST_COVERAGE_THRESHOLD=80
+export TEST_DELTA_THRESHOLD=-10  #current commit test coverage minus last commit on main's coverage
```

Comment:
Now that I think about it, the delta threshold is something that kind of depends on the size of the code bast. Let's not bother getting fancy with it though, haha, it'll just be something to tweak over time.

I think the delta threshold could be lower though, there's not much room for a branch to decrease by 10% of the code base size and not _also_ end up below the coverage threshold of 80% of the code base. Maybe a value of -5 or even lower?

Reply:
I wanted to give a large margin, but what you've mentioned makes sense and have changed it back to -5.